install/docker-extension/ui/src/components/ExtensionComponent/ExtensionComponent.js

```diff
@@ -230,19 +231,25 @@ const ExtensionsComponent = () => {
         ...body,
         application_data: { name, application_file: event.target.result },
       });
-
-      fetch(proxyUrl + "/api/application", {
-        method: "POST",
-        headers: { "Content-Type": "application/x-www-form-urlencoded;charset=UTF-8" },
-        body,
-      })
-        .then((res) => {
-          console.log(res);
-          window.ddClient.desktopUI.toast.success("Compose file has been uploaded with name: " + name);
+      if(!(type.includes(".yaml")||type.includes(".yml"))){
+        window.ddClient.desktopUI.toast.error("Some error occured while uploading the compose file. ")
```

Comment:
adding a `return` statement would prevent it from moving further and executing unwanted lines of code.

Reply:
it is a better way of expressing the intention as well. And you don't have to use `else` 😉

Reply:
Nn this case, it doesn't really matter.

Reply:
Thanks 😄